src/components/TheNavigation.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="navigation-container">
+    <button
+      class="btn-back btn btn-nav"
+      :disabled="isBackBtnDisabled"
+      @click="goBack"
+    >
+      <font-awesome-icon
+        icon="fa-solid fa-angles-left"
+        size="xl"
+        class="icon-left"
+      />
+      Back
+    </button>
+    <button
+      class="btn-forward btn btn-nav"
+      :disabled="isForwardBtnDisabled"
+      @click="goForward"
+    >
+      Forward
+      <font-awesome-icon
+        icon="fa-solid fa-angles-right"
+        class="icon-right"
+        size="xl"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    lastState: {
+      type: Array,
+    },
+
+    id: {
+      type: Number,
+      default: 0,
+    },
+  },
+  data() {
+    return {
+      index: 1,
+      isBackButtonClicked: false,
+      isForwardButtonClicked: false,
+      currentState: null,
+      isBackBtnDisabled: true,
+      isForwardBtnDisabled: true,
+    };
+  },
+  methods: {
+    goBack() {
+      this.isBackButtonClicked = true;
+      this.isForwardButtonClicked = false;
+      this.index--;
+
+      this.lastState.forEach((state) => {
+        state.isActive = false;
+      });
+      this.lastState[this.index].isActive = true;
+    },
+    goForward() {
+      this.isForwardButtonClicked = true;
+      this.isBackButtonClicked = false;
+
+      this.index++;
+      this.lastState.forEach((state) => {
+        state.isActive = false;
+      });
+      this.lastState[this.index].isActive = true;
+    },
+  },
+  watch: {
+    lastState() {
+      if (this.lastState.length > 1) {
+        this.isBackBtnDisabled = false;
+      }
+    },
+    isBackButtonClicked() {
+      if (this.isBackButtonClicked) {
+        this.isForwardBtnDisabled = false;
+      }
+    },
+
+    isForwardButtonClicked() {
+      if (this.isForwardButtonClicked) {
+        this.isBackBtnDisabled = false;
+      }
+    },
+
+    index() {
+      if (this.index === 0) {
+        this.isBackBtnDisabled = true;
+      }
+      if (this.index === this.lastState.length - 1) {
+        this.isForwardBtnDisabled = true;
+      }
+    },
+
+    id() {
+      this.index = this.id;
+    },
+  },
```

Comment:
Dodal sem gumba **Back & Forward**, ki se premikata med odprtimi karticami. Dodal sem `lastState `prop Array, ki glede na klikano kartico pusha, podatke oz object odprte kartice v array.  Glede na `id` kartice določim `index` trenutno odprte kartice in na podlagi tega `indexa  `se premikam med karticami naprej ali nazaj. Če je v array `lastState` več kot en zadetek, enable gumb `Back`. Če kliknem gumb `Back`, enabled gumb `Forward`. Če pridem na začetek arraya, se pravi na prvo odprto kartico, disable Back in obratno za Forward, če pridem do konca array, torej do zadnje odprte kartice, disable Forward.